patterns/hidden-404.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: 404
+ * Slug: twentytwentyfive/hidden-404
+ * Inserter: no
+ *
+ * @package TwentyTwentyFive
+ */
+
+?>
+
+<!-- wp:columns {"style":{"spacing":{"padding":{"top":"50px","bottom":"50px","left":"50px","right":"50px"},"blockGap":{"left":"22px"}}}} -->
+<div class="wp-block-columns" style="padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
+	<!-- wp:column {"width":"40%"} -->
+	<div class="wp-block-column" style="flex-basis:40%">
+		<!-- wp:image {"width":"524px","height":"670px","scale":"cover","sizeSlug":"large","linkDestination":"none"} -->
```

Comment:
I believe we want to use the portrait style for the image, and not hardcoded dimensions.